klock/src/commonMain/kotlin/com/soywiz/klock/KlockLocale.kt

```diff
@@ -30,7 +30,7 @@ abstract class KlockLocale {
         }
     }
 
-    open fun getOrdinalByDay(day: Int, context: KlockLocalContext = KlockLocalContext.empty): String = ordinals[day]
+    open fun getOrdinalByDay(day: Int, context: KlockLocalContext = KlockLocalContext()): String = ordinals[day]
```

Comment:
One minor thing here. Could you create a companion object with something like:

val Default = KlockLocalContext()

That way in the default case it won't allocate on each call

Reply:
Sure, will do.

Also, what do you think about the name `KlockLocalContext`? [Here](https://github.com/korlibs/klock/issues/153#issuecomment-791742614) you've proposed `local` but maybe you've meant `locale` (`KlockLocaleContext`)?

If you decide it's worth renaming, I think it's better to rename it before merging, so I will do it too.

Reply:
Looks either like a typo, or my current self thinks that my other day self was drunk :)

Right now I reviewed it in a phone and I didn't notice the change lol 🤣

My criteria is that if it could be an inner class, it would be named KlockLocale.Context, without the dot KlockLocaleContext.

Local doesn't make sense to me, at least not now 🤷🏻

Reply:
Done